### PR TITLE
Bug fixes and enhancements to HarbourBridge UI

### PR DESCRIPTION
**Pull request by shreyakhajanchi**
- Fixed issue b/267266872 of failure in dump file migration
- Added upload file option in source database connection in case of load by session file
- Remove datastream and dataflow job display components from UI in case of bulk migration

### CODE REVIEW

File: ui/src/app/components/source-details-form/source-details-form.component.ts [hunk 1]
```diff
@@ -54,6 +61,7 @@ export class SourceDetailsFormComponent implements OnInit {
       },
       error: (err: any) => {
         this.errorMsg = err.error
+        console.log(this.errorMsg)
```
Comment (manitgupta) [inline ui/src/app/components/source-details-form/source-details-form.component.ts:64]: Where does this log to?

File: ui/src/app/components/source-details-form/source-details-form.component.html [hunk 2]
```diff
@@ -10,15 +10,23 @@
...
+                <h4 class="primary-header">Load from database dump</h4>
                 Dump File
-                <mat-form-field class="full-width" appearance="outline">
+                <mat-form-field appearance="outline">
                     <mat-label>File path</mat-label>
-                    <input matInput placeholder="File path" name="filePath" type="text" formControlName="filePath" />
+                    <input matInput placeholder="File path" name="filePath" type="text" formControlName="filePath"
+                        readonly (click)="file.click()" />
+
+                    <mat-spinner matSuffix *ngIf="uploadStart && !uploadSuccess && !uploadFail" [diameter]="25">
```
Comment (manitgupta) [inline ui/src/app/components/source-details-form/source-details-form.component.html:20]: Curious: The static `diameter` value of `25`, does it scale automatically when viewing in screens of different sizes?

File: ui/src/app/components/source-details-form/source-details-form.component.ts (hunk 1 above)
Comment (shreyakhajanchi) [reply ui/src/app/components/source-details-form/source-details-form.component.ts]: This was debugging. Removed it from the PR

File: ui/src/app/components/source-details-form/source-details-form.component.html (hunk 2 above)
Comment (shreyakhajanchi) [reply ui/src/app/components/source-details-form/source-details-form.component.html:20]: This doesn't scale based on screen size

**Review by manitgupta — APPROVED** [review]

**Pull request closed without merging**